finetuner-workflow/dataset-downloader/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.19-alpine as source
```

Comment:
Can we switch the `dataset-downloader/Dockerfile` to using `ko`? You can see an example usage of this: here: https://github.com/wbrown/gpt_bpe/blob/main/.github/workflows/main.yml

Reply:
We might want to get gitlab setup to build this/put this in ml-images. @todie, @salanki?

Reply:
Are you suggesting we get a build pipeline set up with this repo or moving the downloader to [the ml containers repo](https://github.com/coreweave/ml-containers) and building it from there?